packages/core/src/models/lessonPlans.ts

```diff
@@ -335,75 +335,4 @@ ${p.content}`,
       WHERE id = ${id}`;
     return result;
   }
-
-  async search(
```

Comment:
This was unused, and the table 'lesson_plans' no longer has embeddings in it, so this will not work.